infra/base-images/base-clang/checkout_build_install_llvm.sh

```diff
@@ -148,7 +150,24 @@ ninja cxx
 ninja install-cxx
 rm -rf $WORK/msan
 
+# DataFlowSanitizer instrumented libraries.
+mkdir -p $WORK/dfsan
+cd $WORK/dfsan
+
+cmake -G "Ninja" \
```

Comment:
actually, let me move this into a separate function, too much duplication

Reply:
+1, these cmake calls look too similar to others in this file.